### PR TITLE
cargo_c_bin, fix some architecture statements

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe [hunk 1]
```diff
@@ -18,8 +18,9 @@ elif [ "$targetArchitecture" = x86_64 ]; then
 	SOURCE_DIR="cargo-c-$srcGitRev/64bit"
 fi
 
-ARCHITECTURES="all !x86_gcc2"
+ARCHITECTURES="!all x86_64"
 SECONDARY_ARCHITECTURES="x86"
+DISABLE_SOURCE_PACKAGE="yes"
```
Comment (Begasus) [inline dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe:23]: Looks correct now @korli ?

Comment (korli) [reply dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe:23]: Looks good!

Comment (korli) [reply dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe:23]: The lint fails: please add an empty SOURCE_URI for all arches.

File: dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe [hunk 2]
```diff
@@ -16,10 +16,13 @@ elif [ "$targetArchitecture" = x86_64 ]; then
 	SOURCE_URI="https://github.com/Begasus/cargo-c/archive/$srcGitRev.tar.gz"
 	CHECKSUM_SHA256="0af0cef06db1aa7d46ca6ce60ad89bc1ca0c3705c51a7f80f6f87e0b2b84d8f7"
 	SOURCE_DIR="cargo-c-$srcGitRev/64bit"
+else
+	SOURCE_URI=""
```
Comment (Begasus) [inline dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe:20]: lint error: list index out of range
But if I put this outside of the if loop I get that error here localy?

Comment (korli) [reply dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe:20]: Might also add CHECKSUM_SHA256

Comment (Begasus) [reply dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe:20]: Didn't solve lint error (works locally) :/

Comment (korli) [reply dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe]: works now

**Pull request merged**